src/object/RideObject.cpp

```diff
@@ -79,6 +80,21 @@ void RideObject::ReadLegacy(IReadObjectContext * context, IStream * stream)
     if (gConfigInterface.select_by_track_type) {
         _legacyType.enabledTrackPieces = 0xFFFFFFFFFFFFFFFF;
     }
+    else
+    {
+        // When not in select by track type mode, add boosters if the track type is eligible
+        for (int i = 0; i < 3; i++) {
+            if (_legacyType.ride_type[i] == RIDE_TYPE_LOOPING_ROLLER_COASTER ||
+                _legacyType.ride_type[i] == RIDE_TYPE_CORKSCREW_ROLLER_COASTER ||
+                _legacyType.ride_type[i] == RIDE_TYPE_TWISTER_ROLLER_COASTER ||
+                _legacyType.ride_type[i] == RIDE_TYPE_VERTICAL_DROP_ROLLER_COASTER ||
+                _legacyType.ride_type[i] == RIDE_TYPE_GIGA_COASTER ||
+                _legacyType.ride_type[i] == RIDE_TYPE_JUNIOR_ROLLER_COASTER) {
```

Comment:
Braces go on separate lines. Line 79 also suffers from this, so feel free to fix that line too.